python/pyarrow/parquet/core.py

```diff
@@ -375,6 +381,13 @@ def num_row_groups(self):
         """
         return self.reader.num_row_groups
 
+    def close(self):
+        self.reader.close()
+
+    @property
+    def closed(self) -> bool:
+        return self.reader.closed
```

Comment:
Does this property exist on ParquetReader? 
(this property also doesn't seem to be tested)

Reply:
Good catch. Added to ParquetReader and test.